core/js/typespecs/api_types.ts

```diff
@@ -37,14 +37,17 @@ import { objectNullish } from "../src/zod_util";
 
 const auditSourcesSchema = z.enum(VALID_SOURCES);
 
+export const spanTypeSchema = z
+  .enum(spanTypeAttributeValues)
```

Comment:
the zod docs say to use `nativeEnum` for typescript enums (https://zod.dev/?id=native-enums). Haven't looked into how it's different from plain old `enum`

Reply:
`spanTypeAttributeValues` is a `string[]`. I think they also recommend zod enums over native enums for schema definitions.